util/normalize.ts

```diff
@@ -0,0 +1,8 @@
+export function normalizeIdent(ident: string) {
```

Comment:
Should we move these into the `codegen` dir?

Reply:
No; these are primarily used not in the codegen but in the metadata preprocessing.